examples/sphere/shallow_water.jl

```diff
@@ -398,7 +398,7 @@ prob = ODEProblem(rhs!, y0, (0.0, T), (f = f, h_s = h_s))
 
 haskey(ENV, "CI_PERF_SKIP_RUN") && exit() # for performance analysis
 
-sol = solve(
+sol = @timev solve(
```

Comment:
Thanks Charlie. If you think that is not that critical to track allocations for the shallow-water test case, then I think we can remove it. I was just wondering yesterday, if that was a somewhat "wasteful" case or not. But I see the [artifact](https://s3.amazonaws.com/buildkiteartifacts.com/52b8ca89-71e4-4c6b-b9a2-94f4dcea2227/28ea22e8-e5a2-4d21-81ab-344cd7c9b38c/721a99d4-d462-4d2a-b291-1849b949701d/d3da8157-70f6-4fbe-abb2-0cf5374e9d60/perf/allocations_output/allocations_shallow_water_barotropic_instability.png?response-content-type=image%2Fpng&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQPCP3C7LQXZWRFXO%2F20220112%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20220112T222729Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEDMaCXVzLWVhc3QtMSJHMEUCIHWyG%2BAAfuW6UEOoY9YdvwtaklZeSXOCYMH3OCehv7KmAiEAlyoz%2BfCjUk2GpjAWGaM3YMMMpOwwrQzb89gXEBrRRLgq%2BgMITBAAGgwwMzIzNzk3MDUzMDMiDE7mVAkbB3nXthJbMyrXAxPkpUQoEjA4PVp4JxXSAIWPaJY2fg1AyI6U8XAnYUee1KIoR5X7TeaFtkhcYIkXfz0HK9VStcMpbjo7U%2BO45BGRrn83Jqoihh%2B6cv50sGrwGY0EQo7vOY%2BCLFrxqCUbSVyo8jN6jEFQS8R3OL797U70CBtv%2FiRNm87EWPoPJgMu9LAfkz%2F2hnCq3CKBh%2BEETd5DgiGVgZpR2u7FA%2FBmZTgXcCn5NeK0wawGHhfl3n4hNlU8XbloJE2M9w72x%2B8FXJpn6Rd5viB9OwJyfOGWwq4aXmeulVTYh2FUz1K1b%2B5shii%2F%2BN3yXK9p5%2B1dDHVLvB%2B7n011piWDZaA6h13Krj94VrJc0qprua9XPsiIRcmcwsYKm8%2Fhoo3adlI3YtaJYtVgwDTz656N6IWANcOYGZ2yN5bcmIuaakSTq2lEMbKe0TJsenDuARLkH%2B820Da8VoWk%2BpBrfcdOqR2pk%2FKvONkttzinUdtSgqZsxxvPxzPFb7%2BWIZvKzOVf0YIB35vEWZ9PRzWq1OTxtbTM9RNGOTvR1rt%2BOFPu4AjI0DRgTKCBrQQGdrc2VkkhSTlfFKjyr%2F9ATQkwQCzeqEBdpxXrHX%2FBHUo%2BCu0Cpnzk%2BMwP%2BDd3pgysU4%2FzEjDYvfyOBjqlAXT%2BpfZyarch9c9n1lfI5ojwpIJUcYlzvdMXPFPyVcN%2FyVEjy7sSBjEE97YlzwtV3kFyPOYh7hJ47OJsNgL0Usoj%2ByxxgJKZd6HSoYLfzKX9yvcn8SAzlpOr%2FhC8npkjcyMlVEyQ9el8nuuHel20izR3yNCVIG%2BS%2F4sL%2B3vVV0rbJzX226DH%2FO4jgjZXnpnLwX9r9W8AKfhjFYx%2FN0WTtBBf0vHgxA%3D%3D&X-Amz-Signature=0c3a0b84fbbc59c5bdfb8c0a10fb727497a08079dfe9fb9452c6d2d40882d9bb) now

Reply:
I think it's fine for now. I'm going to try to keep these perf changes minimal.